components/sup/protocols/event.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
+
+package supervisor.event;
```

Comment:
I'm not sure how much it needs to be the case, but for Automate we're doing java-esque global-y namespaces like `package chef.automate.domain.applications;` I don't think it will hurt anything since there doesn't seem to be a ton of inter-org sharing/re-use in protobufs, but worth considering.

Reply:
That's not a bad idea, actually.

Reply:
@danielsdeleo Check out https://github.com/habitat-sh/habitat/pull/6344/commits/6c288f865beaa10d8e32af9281a16ae1806f7221